src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,52 +1,101 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+
 /**
  * <p>Реалізувати свій ArrayList який імплементує інтерфейс List</p>
  */
 public class ArrayList<T> implements List<T> {
+    private int size;
+    private final int defaultCapacyty = 10;
```

Comment:
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names

Reply:
done

Reply:
[ERROR] /home/travis/build/mate-academy/jv-array-list/src/main/java/core/basesyntax/ArrayList.java:10: Abbreviation in name 'DEFAULT_CAPACITY' must contain no more than '2' consecutive capital letters. [AbbreviationAsWordInName]